>From c7a6432b3ba741fa6fa911e93a7ccc2d0c6b03ed Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Mon, 9 Jul 2012 17:24:20 +0200
Subject: [PATCH 2/3] find_file(): avoid variable-length array

This is a clang compatibility fix.
---
 src/condor_utils/condor_config.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/condor_utils/condor_config.cpp b/src/condor_utils/condor_config.cpp
index c61c53a..cfe806f 100644
--- a/src/condor_utils/condor_config.cpp
+++ b/src/condor_utils/condor_config.cpp
@@ -1107,7 +1107,7 @@ find_file(const char *env_name, const char *file_name)
 	if (!config_source) {
 			// List of condor_config file locations we'll try to open.
 			// As soon as we find one, we'll stop looking.
-		int locations_length = 4;
+		const int locations_length = 4;
 		MyString locations[locations_length];
 			// 1) $HOME/.condor/condor_config
 		struct passwd *pw = getpwuid( geteuid() );
-- 
1.7.7.6

